test.csv

```diff
@@ -1 +1 @@
-column 1,column 2,column 3
1,2,3
+column 1,column 2,column 3
1,2,3
89,99.9,2

```

Comment:
I'm thinking for now we keep just the base values in here for now rather than adding on. When we expand on our testing we can expand on the CSV (and probably add more csvs/other data instances) to test on.